### PR TITLE
Feature contact modal box

### PR DESCRIPTION
**Pull request by doanyennhi**
Please check the responsive design of the modal box and comment if you think the design should be improved

- Implemented page dim feature
- Implemented overlay window and modal boxes with members' info
- Implemented opening and closing the modal box features

### CODE REVIEW

File: public/css/mall-about-us.css [hunk 1]
```diff
@@ -1,3 +1,53 @@
+/** MODAL WINDOW STYLING **/
```
Comment (miketvo) [inline public/css/mall-about-us.css:1]: Very good commenting

File: public/js/about-us.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+function toggleModalBox(modalBoxId) {
+    const PAGEDIM = document.getElementById("dimmed-page");
+    const OVERLAY = document.getElementById("overlay-modal-window");
+    let modalBox = document.getElementById(modalBoxId);
```
Comment (miketvo) [inline public/js/about-us.js:4]: this should be a const. E.g.
```javascript
const MODAL_BOX = document.getElementById(modalBoxId);
```

**Review by miketvo — CHANGES_REQUESTED** [review]

**Comment by miketvo** [comment]
@doanyennhi 

Good code! However I can't close the modal boxes. Please check on that issue.

**Review by miketvo — APPROVED** [review]
Very good attention to detail.

**Review by rider3458 — APPROVED** [review]

**Pull request merged**